source/clear-linux/tutorials/web-server-install/web-server-install.rst

```diff
@@ -1,106 +1,92 @@
 .. _web-server-install:
 
-Creating a Clear Linux based web server
+Create a Clear Linux based web server
 #######################################
 
-Prerequisites
-*************
+This tutorial shows you how to create a LAMP server on |CL| and how to use phpMyAdmin to manage an associated database.
 
-In order to create a web server using |CL| as the host OS your host
-system must be running |CL|. Therefore, this tutorial assumes you have
-already gone through the :ref:`bare metal installation<bare-metal-install>`
-instructions.
+In order to create a web server using |CL| as the host OS, your host system must be running |CL|. This tutorial assumes you have successfully installed :ref:`Clear Linux on bare metal<bare-metal-install>`.
 
-Before installing any new packages, update the |CL| OS with the
-console command:
+Before you install new packages, update the |CL| OS with the following console command:
 
-.. code-block:: console
+.. code-block:: bash
 
    sudo swupd update
 
-Creating a LAMP Server
+Create a LAMP Server
 **********************
 
-A server with Linux\*, Apache\*, MySQL\*, and PHP\* installed is known as a
-LAMP server, allows you to set up a fully functional web server, and enables
-you to host your own website. This tutorial walks you through the process of
-creating a LAMP server by installing and configuring each component. In place
-of MySQL, we will be installing MariaDB which is a drop- in replacement for
-MySQL. Once the LAMP server component installations are complete, we add
-phpMyAdmin to manage your MariaDB databases.
+A LAMP server uses Linux\*, Apache\*, MySQL\*, and PHP\* to set up a fully functional web server and host a website. Note that this tutorial installs MariaDB, which is a drop-in replacement for MySQL. 
+
+This tutorial follows these steps:
 
-Installing Apache
+1. Install Apache.
+2. Change the default configuration and data directory.
+3. Install PHP.
+4. Install MariaDB.
+5. Install phpMyAdmin.
+6. Set up a WordPress database.
+
```

Comment:
.rst convention is to use '#.' preceding numbered steps. Please change these to reflect that. 

- Revise your Step 5 to: "Install phpMyAdmin and create a database"
- Omit your Step 6 above.
- Remember: The goal of this tutorial is to create a 'WordPress-ready" database. 

See "Lists and Quote-like blocks": 
http://www.sphinx-doc.org/en/stable/rest.html#lists-and-quote-like-blocks

I  recommend bookmarking this page for reference: http://www.sphinx-doc.org/en/stable/index.html